Library/Homebrew/test/test_patching.rb

```diff
@@ -122,7 +122,9 @@ def test_single_patch_dsl_with_incorrect_strip
             url PATCH_URL_A
             sha256 PATCH_A_SHA256
           end
-        end.brew(&:patch)
+        end.brew do |f, _staging|
+          f.patch
+        end
```

Comment:
Well, the GitHub Ruby Style Guide says to use `{ ... }` for single-line blocks, but to avoid multiline chaining like this entirely, and to refactor a multi-line chain in to methods. (It's in the bullet point that starts with "Prefer {...} over do...end for single-line blocks".)

Since the multi-line chain was there before, and I'm not sure how to refactor it away without (imho) hurting readability, I'm going to switch this over to `{ ... }` and leave it as a chain. More thorough style fixes can be done in another PR; there's a bunch of other items that `brew style` also complains about in this file.